docs/threat-model.md

```diff
@@ -0,0 +1,553 @@
+- **Integrity**: MEDIUM - Resource exhaustion could compromise node operation and validation
+- **Availability**: HIGH - Excessive resource usage could force SPOs offline or prevent participation
+
+**Leios-Specific Considerations**: New responsibilities (EB creation, voting, additional network protocols) must not significantly increase SPO operational costs relative to incentives or create barriers to participation.
+
+### 2.5 Decentralization Properties
+**Description**: The distribution of block production, voting power, and network participation across many independent operators.
+
+**CIA Impact:**
+- **Confidentiality**: LOW - Centralization patterns are observable but don't directly affect data secrecy
+- **Integrity**: HIGH - Centralization increases risk of coordinated attacks on consensus
+- **Availability**: HIGH - Centralized infrastructure creates single points of failure
+
+### 2.6 High Throughput
+**Description**: The enhanced transaction processing capacity that Leios provides beyond basic Praos liveness, enabling the network to handle significantly more transactions per unit time.
+
+**CIA Impact:**
+- **Confidentiality**: Not applicable - throughput is a performance metric
+- **Integrity**: LOW - Reduced throughput doesn't corrupt data but may affect user expectations
+- **Availability**: HIGH - Throughput reduction defeats the primary purpose of the Leios upgrade
+
+**Leios-Specific Considerations**: EB certification failures, voting delays, or resource exhaustion attacks directly impact the throughput gains Leios is designed to provide.
+
+## 3. Threats
+
+Notable threats to the system that could impact assets.
+
+### 3.1 Network-Level Threats
+
+#### T1: Mempool Partitioning Attack
+**Description**: Attacker deliberately partitions the mempools of block producing nodes by submitting conflicting transactions (spending the same inputs) to different network segments, creating inconsistent views of valid transactions across the network.
+
+**Prerequisites**:
+- Control over multiple network positions to segment the peer-to-peer network
+- Ability to create valid but conflicting transactions (same inputs, different outputs)
+- Knowledge of network topology and SPO peer relationships
+- Timing coordination to submit conflicts before natural mempool synchronization
+
+**Attack Vector**:
+1. Attacker maps network topology and identifies cluster boundaries
+2. Creates conflicting transaction pairs spending identical UTXOs
+3. Submits Transaction A to Network Segment 1, Transaction B to Network Segment 2
+4. Uses network position control (BGP, routing, eclipse techniques) to prevent cross-segment propagation
+5. SPOs in different segments create EBs endorsing different conflicting transactions
+6. Voting nodes must choose between conflicting EBs, potentially causing certification failures
+
+**Cost**: MEDIUM-HIGH - Requires significant network infrastructure, multiple nodes, and sustained coordination
```

Comment:
```suggestion
**Cost**: MEDIUM - Requires significant network infrastructure, multiple nodes, and sustained coordination
```
This doesn't require deploying many nodes if they are nodes custom-modified for this attack. The attack would require more nodes if vanilla ones are used.